src/core/border_router/routing_manager.hpp

```diff
@@ -212,6 +212,7 @@ class RoutingManager : public InstanceLocator
 
     static constexpr uint32_t kDefaultOmrPrefixLifetime    = 1800; // The default OMR prefix valid lifetime. In sec.
     static constexpr uint32_t kDefaultOnLinkPrefixLifetime = 1800; // The default on-link prefix valid lifetime. In sec.
+    static constexpr uint32_t kDefaultNat64PrefixLifetime  = 60;   // The default NAT64 prefix valid lifetime. In sec.
```

Comment:
is this value defined in any spec ?

Reply:
No, the spec doesn't specify the lifetime/frequency.

Reply:
I feel 60s is too short. Maybe better use a longer duration, e.g. 300.